app/src/main/res/layout/qsb_container.xml

```diff
@@ -1,30 +1,59 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
-     Copyright (C) 2016 The Android Open Source Project
+<xyz.klinker.blur.launcher3.QsbContainerView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we add `16dp` of padding to this `QsbContainerView`? The pill widget seem to be just a little to high, to me.